### PR TITLE
Add instrument methods

### LINKED ISSUE #281: Add instrument methods, docs, and tests
- [ ] Export Instruments (Data Entry Forms)
- [ ] Export PDF file of Instruments
- [ ] Import Instrument-Event Mappings

### PR DESCRIPTION
**Pull request by patking02**
Fixes #281

### CODE REVIEW

**Review by pwildenhain — COMMENTED** [review]
Nicely done again 👏🏻 

See https://github.com/patking02/PyCap/pull/2 for a few quick fixes

Did you want me to cut another release as soon as this is merged? Or did you have more contributions you want to add in the short term?

File: redcap/methods/instruments.py [hunk 1]
```diff
@@ -20,6 +11,104 @@
...
+        if record:
+            payload["record"] = record
+        if event:
+            payload["event"] = event
+        if instrument:
+            payload["instrument"] = instrument
+        if repeat_instance:
+            payload["repeat_instance"] = str(repeat_instance)
+        if all_records:
+            payload["allRecords"] = str(all_records == "True")
```
[the remark this answers is not in the document]
Comment (patking02) [reply redcap/methods/instruments.py:91]: Yeah, not sure what I was thinking there. I think I was hung up by the wording in the API documentation for Export PDF and the `allRecords` option and was trying to work around that...

I'll adjust my PR...

**Comment by patking02** [comment]
> Nicely done again 👏🏻
> 
> See [patking02#2](https://github.com/patking02/PyCap/pull/2) for a few quick fixes
> 
> Did you want me to cut another release as soon as this is merged? Or did you have more contributions you want to add in the short term?

No more contributions for the moment! Happy for you to cut another release if that is no problem!

**Comment by pwildenhain** [comment]
> > Nicely done again 👏🏻
> > See [patking02#2](https://github.com/patking02/PyCap/pull/2) for a few quick fixes
> > Did you want me to cut another release as soon as this is merged? Or did you have more contributions you want to add in the short term?
> 
> No more contributions for the moment! Happy for you to cut another release if that is no problem!

New release is up ❤️ https://pypi.org/project/pycap/

**Pull request merged**